features/works/work_edit_multiple.feature

```diff
@@ -138,3 +138,16 @@ Feature: Edit Multiple Works
     Then I should not see "coauthor" within ".byline"
     When I view the work "Shared Work 2"
     Then I should not see "coauthor" within ".byline"
+
+  Scenario: User applies a private work skin to multiple coauthored works
+    Given the following activated users with private work skins
+      | login       |
+      | lead_author |
+      | coauthor    |
+    And I am logged in as "lead_author"
```

Comment:
Could you indent the `And`s two spaces deeper than the `Given`s, `When`s, and `Then`s?

Reply:
Fixed.